croissant-rdf/pyproject.toml

```diff
@@ -25,8 +25,7 @@ dependencies = [
     "kaggle >=1.6.17",
     "openml >=0.15.1",
     "rich >=13.9.4",
```

Comment:
nit: Remove rich in favour of tqdm as pointed in the initial PR?

Reply:
We started with tqdm but moved to rich because it's... richer ;) Prefer we keep it otherwise we're sort of thrashing. https://medium.com/pythoneers/from-tqdm-to-rich-my-quest-for-better-progress-bars-afff39985ffc